task10-vue/src/components/ObjectSection.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <article className="article article--mist">
+    <div className="article__container article__container--mist">
+      <h3 className="article__title article__title--mist">Mist Objects</h3>
+      <a href="#" className="article__link article__link--mist">See all</a>
+    </div>
+    <div className="article__flex article__flex--mist">
+      <MysticObjectCard
+        v-for="mistObject in createObjects(4)"
```

Comment:
4 should be a prop, in that case, you could use "window.innerWidth" in order to send 3 as a prop on mobile, 4 otherwise